### PR TITLE
Tag Playground v0.2.0 [https://github.com/Rory-Finnegan/Playground.jl]

### PR DESCRIPTION
**Pull request by Rory-Finnegan**
I realize it has taken me a while to update Playground to 0.5, but I was hoping a couple of my dependencies would get updated. I've updated the minor version number as I've dropped support for DeclarativePackages.

### CODE REVIEW

**Comment by andreasnoack** [comment]
You might want to consider https://github.com/attobot/attobot

File: Playground/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,5 @@
+julia 0.4
+ArgParse
+YAML
+Mocking
+Compat
```
Comment (tkelman) [inline Playground/versions/0.2.0/requires]: It's best to declare a minimum version dependency that implements the features you need.

Comment (Rory-Finnegan) [inline Playground/versions/0.2.0/requires]: Good point. I'm guessing I can make those changes and tag a new patch release?

Comment (tkelman) [inline Playground/versions/0.2.0/requires]: no, tightening bounds requires adjusting the requires file of the previous tags in-place in metadata. making the change in require will be good before the next tag though.

Comment (Rory-Finnegan) [inline Playground/versions/0.2.0/requires]: Alright, will do. Thanks.

**Pull request merged**